packages/python/m/cli/commands/github/__init__.py

```diff
@@ -11,4 +13,49 @@
     'description': cleandoc(_DESC),
 }
 
+def repo_and_owner():
```

Comment:
I like the idea. Only suggestion is to move this function from `m/cli` to `m/github`.

Haven't looked into the tests but I think they may be failing due to `m` giving the `m` repo based on `m`.
So some tests will need to do some mocking of `get_remote_url` to provide a valid git url.

Reply:
oh, i think i just realized this is not being used. The one that is doing the magic is the `repo_and_owner` from `m.github.arguments`. I think that works. This file can be reverted and all the other command arguments can extend from it.

By the way, go ahead and pull the latest code since we got pydantic 2. See https://github.com/jmlopez-rod/m/pull/91